UI improvements

calharden opened the pull request.

calharden committed:
Updated npm dependencies

calharden committed:
Made several UI improvements

calharden committed:
Updated NuGet packages

calharden committed:
Fixed minor display issues on reply and response forms

JacobGralinski reviewed (changes requested):
- On the response example page, the text at the top is wrong.
- The color of the header is too dark and the logo should not be centered, it looks weird.
- The footer appears too small.
- The Back buttons should not be gray as they look like they are not clickable.
- There is a lot of excess white space after text, especially the Let's Get Started page.
- The Continue button on the home page is way too wide.
- Getting rid of the brief number in the My Briefs page was unnecessary.
- Updating to Bootstrap 4 was unnecessary
- This is way too many changes for the "minor changes" Colleen is expecting.

calharden committed:
Fixed text on response example page, aligned state bar logo to left, made footer larger, modifed continue button on home screen.

calharden commented:
> On the response example page, the text at the top is wrong.
- Fixed the text on the response example page in last commit.
> The color of the header is too dark and the logo should not be centered, it looks weird.
- The color of the header is actually lighter than the background color of the current header, If you have another color in mind I can change it.
> The footer appears too small.
- Made the footer about the same size as the current one in last commit.
> The Back buttons should not be gray as they look like they are not clickable.
- What color do you think the buttons should be? I thought they looked kind of weird when they were both blue. [Possible Options](https://getbootstrap.com/docs/4.0/components/buttons/#examples), [More options](https://getbootstrap.com/docs/4.0/components/buttons/#outline-buttons)
> There is a lot of excess white space after text, especially the Let's Get Started page.
- I removed a bunch of excess white-space from several of the pages while making the changes. There is less white-space on the Lets Get Started Page then there was before. Could you post a screenshot of any white-space looks excessive in the slack?
> The Continue button on the home page is way too wide.
- I made the continue button normal sized and aligned it to the right in my last commit.
> Getting rid of the brief number in the My Briefs page was unnecessary.
- I got rid of the brief number because I thought showing the type of brief was more valuable and the spacing looked weird when both of them were there.
> Updating to Bootstrap 4 was unnecessary
- I moved us to bootstrap 4 because it made it easier to make the various account pages look good and it allowed me to add a confirmation dialog to the delete button on the My Briefs page.
> This is way too many changes for the "minor changes" Colleen is expecting.
- The only major changes was splitting the login-register page into two separate pages. Most of the other were fairly minor, such changing the location of buttons slightly, fixing a few grammar errors, fixing an exception on the initial brief form, or changing the size of text (such as moving from an h2 to an h4). Its just several minor changes and it looks larger than it actually is on the diff page because I reformatted a couple of files.

calharden committed:
Changed all back buttons to btn-dark

calharden committed:
Changed links in legal resources page to match the style on rest of the site.

calharden committed:
Removed unneeded css files

JacobGralinski committed:
Added my name to the Privacy Policy for the future.

JacobGralinski committed:
Merge branch 'ui-improvements' of https://github.com/msoeSE/legalBrief into ui-improvements
